### PR TITLE
[no ticket][risk=no] Puppeteer e2e: Don't use relative paths in module imports

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "private": true,
   "license": "BSD",
   "scripts": {
-    "test": "cross-env WORKBENCH_ENV=dev jest --detectOpenHandles",
+    "test": "cross-env WORKBENCH_ENV=dev jest",
```
Comment (aweng98) [inline e2e/package.json:9]: unrelated to imports. It's a small change to run tests in parallel for testing on localhost.

**Comment by aweng98** [comment]
@jaycarlton hi, one of your last PR comment inspired this PR. Can you review and approve? Thanks

File: e2e/tests/workspace/workspace-ui.spec.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-import HomePage from '../../app/home-page';
-import WorkspaceCard from '../../app/workspace-card';
-import WorkspacesPage from '../../app/workspaces-page';
-import {signIn} from '../app';
+import BaseElement from 'app/aou-elements/base-element';
+import {SideNavLink} from 'app/authenticated-page';
+import HomePage from 'app/home-page';
+import WorkspaceCard from 'app/workspace-card';
+import WorkspacesPage from 'app/workspaces-page';
+import {signIn} from 'tests/app';
```
Comment (jaycarlton) [inline e2e/tests/workspace/workspace-ui.spec.ts:6]: Is there a way to set it up so that there's only one `app` directory?

Comment (aweng98) [reply e2e/tests/workspace/workspace-ui.spec.ts:6]: are you talking about `tests/app`? it is a file `app.ts`. I see now filename and path `app` are confusing. I'll rename file `app.ts` in future PR. thank you.

Comment (jaycarlton) [reply e2e/tests/workspace/workspace-ui.spec.ts:6]: Yes, that makes sense.

**Pull request merged**